monitoring/prometheus/generate-prom.js

```diff
@@ -43,53 +42,74 @@ scrape_configs:
     static_configs:
       - targets: ['localhost:9090']
 
- # monitor docker-compose local setups
+  # monitor data growth
 
-  - job_name: 'local-discovery-provider'
-    metrics_path: '/prometheus_metrics'
+  - job_name: 'load-test-census-stage'
+    metrics_path: '/metrics'
     static_configs:
-      - targets: ['host.docker.internal:5000']
+      - targets: ['host.docker.internal:8001']
         labels:
           host: 'host.docker.internal'
-          environment: 'remote-dev'
+          environment: 'stage'
           service: 'audius'
           component: 'discover-provider'
+          job: 'census'
 
-  - job_name: 'load-test-populate'
+  - job_name: 'load-test-census-prod'
     metrics_path: '/metrics'
+    scrape_interval: 30m
     static_configs:
-      - targets: ['host.docker.internal:8000']
+      - targets: ['host.docker.internal:8002']
         labels:
           host: 'host.docker.internal'
-          environment: 'load-test'
+          environment: 'prod'
           service: 'audius'
           component: 'discover-provider'
-          job: 'populate'
+          job: 'census'
+  `)
 
-  - job_name: 'load-test-census-stage'
-    metrics_path: '/metrics'
+  if (process.env.PROM_ENV === "local") {
+    stream.write(`
+  # monitor docker-compose local setups
+
+  - job_name: 'local-discovery-provider'
+    metrics_path: '/prometheus_metrics'
     static_configs:
-      - targets: ['host.docker.internal:8001']
+      - targets: ['host.docker.internal:5000']
         labels:
           host: 'host.docker.internal'
-          environment: 'stage'
+          environment: 'remote-dev'
           service: 'audius'
           component: 'discover-provider'
-          job: 'census'
 
-  - job_name: 'load-test-census-prod'
+  - job_name: 'local-creator-node'
+    metrics_path: '/prometheus_metrics'
+    static_configs:
+      - targets: ['host.docker.internal:3000']
```

Comment:
?
local CNs run on ports 4000, 4001, 4002, and 4003

Reply:
I thought it was :3000 for some reason, but +1 on us also having multiple instances.

That was a new bit that hadn't been tested since I hadn't turned on the new creator node code yet, but I wanted to make sure we had an easily identifiable place to edit when we started working on the integration pieces.

Reply:
cool - i updated this in my PR - pls take a look!
https://github.com/AudiusProject/audius-protocol/pull/3266